tasks/main.yml

```diff
@@ -58,16 +60,19 @@
     src: manage_certs.conf
     dest: "{{ LOAD_BALANCER_MANAGE_CERTS_CONF }}"
 
-- name: copy config watcher systemd service file
+- name: copy config and cert watcher systemd service files
   template:
-    src: haproxy-config-watcher.service
-    dest: /lib/systemd/system/haproxy-config-watcher.service
-
-- name: enable and start config watcher
-  command: systemctl {{ item }} haproxy-config-watcher
+    src: "{{ item }}"
+    dest: "/lib/systemd/system/{{ item }}"
   with_items:
-    - enable
-    - restart
+    - haproxy-config-watcher.service
+    - cert-watcher.service
+
+- name: enable and start config and cert watcher
+  command: systemctl {{ item | join(" ") }}
```

Comment:
Too bad we can't use [ansible systemd](https://docs.ansible.com/ansible/systemd_module.html) here, but it requires ansible 2.2, and we're only [using 2.0.1](https://github.com/open-craft/deployment-deploy-services/blob/master/requirements.txt#L1).